src/Microsoft.PowerShell.Commands.Management/singleshell/installer/MshManagementMshSnapin.cs

```diff
@@ -7,14 +7,10 @@
 namespace Microsoft.PowerShell
 {
     /// <summary>
-    ///
     /// MshManagementMshSnapin (or MshManagementMshSnapinInstaller) is a class for facilitating registry
     /// of necessary information for monad management mshsnapin.
-    ///
     /// This class will be built with monad management dll
```

Comment:
We should just remove this line, doesn't add value

Reply:
Removed.